run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
+
+#set the max memory
+BYTES_PER_MEG=$((1024*1024))
+BYTES_PER_GIG=$((1024*${BYTES_PER_MEG}))
+
+DEFAULT_MIN=$((64 * $BYTES_PER_MEG)) #This is a guess
+regex='^([[:digit:]]+)([GgMm])?i?$'
+num_regex='^[[:digit:]]+'
+unit_regex='[GgMm]'
+
+export NODE_OPTIONS=""
+
+if [ 1 -eq $(echo "${OCP_AUTH_PROXY_MEMORY_LIMIT:-}" | grep -Ec $regex) ] ; then
```

Comment:
I don't have the luxury of using bash regex as best I can tell.  It's a 'dash' interpreter